src/Extensions/SiteTreeSubsites.php

```diff
@@ -267,12 +268,13 @@ public function duplicateToSubsite($subsiteID = null, $includeChildren = false)
      * It may be that some relations are not diostinct to sub site so can stay
      * whereas others may need to be duplicated
      *
-     * @deprecated 2.2.0 Use the "cascade_duplicates" config API instead
+     * This was originally deprecated - Use the "cascade_duplicates" config API instead
```

Comment:
Un-deprecated because it's too risky to implement this as part of this issue - have raised a new issue to look at redeprecating https://github.com/silverstripe/silverstripe-subsites/issues/494